src/SFA.DAS.EmployerIncentives/Configuration/AzureManagedIdentityApiConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Configuration
+{
+    public class AzureManagedIdentityApiConfiguration
```

Comment:
This should be the same as the above - the Url isn't required - its only the method of obtaining the token that is different